gossip/gossip.go

```diff
@@ -777,14 +781,14 @@ func (g *Gossip) bootstrap() {
 			select {
 			case <-bootstrapTimer.C:
 				bootstrapTimer.Read = true
-				// continue
+				// break
```

Comment:
The intention wasn't to continue. Bad choice of words. "continue" just meant proceed to next select statement.